python/tvm/relay/backend/contrib/ethosu/te/pooling.py

```diff
@@ -169,7 +169,7 @@ def pooling_compute(
         [1, 0, 0, 0, 0, 0],
         [0, 1, 0, 0, 0, 0],
         [0, 0, 0, 1, 0, 0],
-        [0, 0, 16, 0, 1, -16],
+        [0, 0, 0, 0, 0, int(ofm_channels)],
```

Comment:
Is `int(...)` necessary here?

Reply:
Yes, since the type of `ofm_channels` is `IntImm`, so without the cast it would end up causing an error in `propagator.py`